### DISCUSSION TITLE
Is it possible to remove "posts" from url?

### DISCUSSION THREAD

**Discussion by dennys** [chipzoller/hugo-clarity#323] [Q&A] [answered]
The url likes the folowing format.

http://localhost:1313/posts/202203/xxxx/

Is it possible to remove "posts" from url? Like the following sample.

http://localhost:1313/202203/xxxx/

**Comment by rootwork** [comment, marked as answer]
You should just be able to move the files from `content/posts/202203/xxxx` to `content/202203/xxxx` and Hugo will update the paths. On my personal site running Hugo Clarity I have posts in `content/blog` rather than `content/posts`.

**Comment by rootwork** [comment]
Also see https://gohugo.io/content-management/urls/ and https://gohugo.io/content-management/organization/ for Hugo's documentation in this area.

**Reply by dennys** [reply]
Hi, I read the document and find another solution, I can add this config in config.toml, thanks.

    posts = ":year/:month/:slug"

**Comment by dennys** [comment]
It works. I'm a newbie for Hugo, I use other theme (Loveit) and the url generated by Loveit is not the same.
I'll read the document to study the detail, thanks.
